orc8r/gateway/python/magma/common/redis/BUILD.bazel

```diff
@@ -0,0 +1,23 @@
+load("@rules_python//python:defs.bzl", "py_library")
```

Comment:
For all of these BUILD files please add the license header :) (See any other BUILD.bazel under lte/gateway/c for an example)

Reply:
I have added the headers.